FirebaseDynamicLinks/Sources/Utilities/FDLUtilities.m

```diff
@@ -295,7 +295,10 @@ BOOL FIRDLMatchesShortLinkFormat(NSURL *URL) {
   // Path cannot be prefixed with /link/dismiss
   BOOL isDismiss = [[URL.path lowercaseString] hasPrefix:@"/link/dismiss"];
 
-  return hasPathOrCustomDomain && !isDismiss && canParse;
+  BOOL matchesRegularExpression =
+      ([URL.path rangeOfString:@"/[^/]+" options:NSRegularExpressionSearch].location != NSNotFound);
```

Comment:
I saw `/[^/]+` pattern was used previously int he SDK. Do you think it is worth adding a comment for what this pattern means?

Reply:
Based on my understanding, this regex is checking for a "/" and should not be having anymore succeeding "/"'s. I'll add it as a comment there.

Reply:
Done